.github/workflows/maven-build.yml

```diff
@@ -78,21 +97,19 @@ jobs:
                 }
               }
             ]
-      - name: Print settings.xml
+
+      # Print settings.xml
+      - name: 🔘 Print settings.xml
         run: cat /home/runner/.m2/settings.xml
-      - name: Build with Maven
+
+      # Build with Maven
+      - name: 📦 Build with Maven
         run: mvn --batch-mode clean package
```

Comment:
no sonar:sonar?

Reply:
no sonar until we finished with generic and setup api-extender project